src/actions/pause/commands.js

```diff
@@ -38,7 +38,7 @@ export function command(type: CommandType) {
  */
 export function stepIn() {
   return ({ dispatch, getState }: ThunkArgs) => {
-    if (getPause(getState())) {
```

Comment:
most of the change is going to `isPaused` from an indirect check to see if `pauseInfo` exists